tests/unit/onyxClearNativeStorageTest.js

```diff
@@ -94,15 +92,70 @@ describe('Set data while storage is clearing', () => {
         Onyx.clear();
         return waitForPromisesToResolve()
             .then(() => {
-                // Then the value in Onyx and the cache is the default key state
+                // Then the value in Onyx, the cache, and Storage is the default key state
                 expect(onyxValue).toBe(DEFAULT_VALUE);
                 const cachedValue = cache.getValue(ONYX_KEYS.DEFAULT_KEY);
                 expect(cachedValue).toBe(DEFAULT_VALUE);
-                const storedValue = Storage.getItem(ONYX_KEYS.DEFAULT_KEY);
+                return Storage.getItem(ONYX_KEYS.DEFAULT_KEY)
+                    .then(storedValue => expect(parseInt(storedValue, 10)).toBe(DEFAULT_VALUE));
```

Comment:
Apparently, it was because:
https://github.com/Expensify/react-native-onyx/blob/335aca03d1c100f013bd3d042399e4dd25e4a066/tests/unit/onyxClearNativeStorageTest.js#L10

Reply:
That seems strange and something we should fix in the mock... by NAB